packages/pages/src/vite-plugin/build/closeBundle/moduleLoader.ts

```diff
@@ -62,6 +67,40 @@ const validateUniqueFeatureName = (
     });
 };
 
+const validateBundles = (serverBundlePaths: string[]) => {
```

Comment:
The server bundle paths aren't the only assets included in the plugin. Everything in the static and renderer folder are also included (see the ci.json file in the starter or the artifact output )